docs/usage/configuration-options.md

```diff
@@ -2517,7 +2514,15 @@ The managers listed above all have this default registryAlias:
 }
 ```
 
-Alias values must be properly formatted URIs.
```

Comment:
Did you really mean to remove this warning? Do we now accept badly formatted URIs?

> Alias values must be properly formatted URIs.

Reply:
Yes, because for `gitlabci` the values are not URIs but just registry prefixes (e.g. `registry.gitlab.com`)

Reply:
we should add links to supported managers/ datasources and add specific samples there

Reply:
Good idea, done in my last commits!